bug/snapshot.go

```diff
@@ -87,5 +92,29 @@ func (snap *Snapshot) addParticipant(participant identity.Interface) {
 	snap.Participants = append(snap.Participants, participant)
 }
 
+// HasParticipant return true if the id is a participant
+func (snap *Snapshot) HasParticipant(id string) bool {
+	for _, p := range snap.Participants {
+		if p.Id() == id {
+			return true
+		}
+	}
+	return false
+}
+
+// HasAnyParticipant return true if one of the ids is a participant
+func (snap *Snapshot) HasAnyParticipant(ids ...string) bool {
+	if len(ids) == 0 {
```

Comment:
you don't need that, if the len is 0, the for loop just doesn't execute

Reply:
Done !